pkg/apiserver/registry/core/namespacedcloudprofile/cloudprofile_suite_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package namespacedcloudprofile_test
```

Comment:
Shouldn't the filename be `namespacedcloudprofile_suite_test.go`?

Reply:
Jup, changed in https://github.com/gardener/gardener/pull/9440/commits/026d5bdc62ce06a524e6839f6d663a7dcf36a6e6